pkg/api/pod/warnings.go

```diff
@@ -151,6 +152,9 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		if v.Glusterfs != nil {
 			warnings = append(warnings, fmt.Sprintf("%s: deprecated in v1.25, this feature will be removed soon after in a subsequent release", fieldPath.Child("spec", "volumes").Index(i).Child("glusterfs")))
 		}
+		if v.Ephemeral != nil {
```

Comment:
make sure VolumeClaimTemplate is not null as well

Reply:
Done.

And fix the import verify failure.